### PR TITLE
Add support for IPv6

### PR DESCRIPTION
**Pull request by Smartappli**
Fixes https://github.com/serge-chat/serge/issues/652

### CODE REVIEW

**Comment by gaby** [comment]
@Smartappli We need to migrate to hypercorn, else we will be running 2 servers at the same time

**Comment by Smartappli** [comment]
> @Smartappli We need to migrate to hypercorn, else we will be running 2 servers at the same time.
> 
> See here: [encode/uvicorn#1529 (reply in thread)](https://github.com/encode/uvicorn/discussions/1529#discussioncomment-7264527)

thanks :)

File: scripts/deploy.sh [hunk 1]
```diff
@@ -26,7 +26,11 @@ if [ "$cpu_arch" = "aarch64" ]; then
 else
 	# Use @jllllll provided wheels
 	cpu_feature=$(detect_cpu_features)
-	pip_command="python -m pip install -v llama-cpp-python==$LLAMA_PYTHON_VERSION --only-binary=:all: --extra-index-url=https://jllllll.github.io/llama-cpp-python-cuBLAS-wheels/$cpu_feature/cpu"
+	if [ "$SERGE_GPU_SUPPORT" = false ]; then
```
Comment (gaby) [inline scripts/deploy.sh:29]: Why is this here?

File: scripts/serge.env [hunk 2]
```diff
@@ -1 +1,2 @@
 LLAMA_PYTHON_VERSION=0.2.26
+SERGE_GPU_SUPPORT=false
```
Comment (gaby) [inline scripts/serge.env:2]: Why is this here?

File: scripts/dev.sh [hunk 3]
```diff
@@ -25,8 +25,12 @@ if [ "$cpu_arch" = "aarch64" ]; then
 	pip_command="python -m pip install -v llama-cpp-python==$LLAMA_PYTHON_VERSION --only-binary=:all: --extra-index-url=https://gaby.github.io/arm64-wheels/"
 else
 	# Use @jllllll provided wheels
-	cpu_feature=$(detect_cpu_features)
-	pip_command="python -m pip install -v llama-cpp-python==$LLAMA_PYTHON_VERSION --only-binary=:all: --extra-index-url=https://jllllll.github.io/llama-cpp-python-cuBLAS-wheels/$cpu_feature/cpu"
+ 	cpu_feature=$(detect_cpu_features)
+	if [ "$SERGE_GPU_SUPPORT" = false ]; then
```
Comment (gaby) [inline scripts/dev.sh:29]: Same here

**Review by gaby — CHANGES_REQUESTED** [review]

File: scripts/deploy.sh [hunk 4]
```diff
@@ -50,7 +50,7 @@ redis_process=$!
 
 # Start the API
 cd /usr/src/app/api || exit 1
-uvicorn src.serge.main:app --host 0.0.0.0 --port 8008 || {
+hypercorn src.serge.main:app --bind 0.0.0.0:8008 --bind [::]:8008 || {
```
Comment (gaby) [inline scripts/deploy.sh:53]: This is forcing ipv6 to everyone. It needs to be an env option.

`SERGE_ENABLE_IPV6`

File: scripts/dev.sh [hunk 5]
```diff
@@ -58,7 +58,7 @@ npm run dev -- --host 0.0.0.0 --port 8008 &
 
 # Start the API
 cd /usr/src/app/api || exit 1
-uvicorn src.serge.main:api_app --reload --host 0.0.0.0 --port 9124 --root-path /api/ || {
+hypercorn src.serge.main:api_app --reload --bind 0.0.0.0:9124 --bind [::]:9124 --root-path /api/ || {
```
Comment (gaby) [inline scripts/dev.sh:61]: Same here as above

**Review by gaby — CHANGES_REQUESTED** [review]

**Comment by gaby** [comment]
@Smartappli Can you take a look at the merge conflicts

**Review by gaby — APPROVED** [review]

**Pull request merged**